files/zh-cn/web/api/filesystemwritablefilestream/truncate/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: FileSystemWritableFileStream：truncate() 方法
+slug: Web/API/FileSystemWritableFileStream/truncate
+---
+
+{{securecontext_header}}{{APIRef("File System API")}}
+
+{{domxref("FileSystemWritableFileStream")}} 接口的 **`truncate()`** 方法用于将与流相关联的文件调整为指定字节大小。
+
+如果指定的大小大于文件当前的大小，文件会被用空字节补充。
+
+调用 `truncate()` 方法同时也会更新文件的指针。如果偏移小于大小，指针会保留原位。如果偏移大于大小，偏移会被设为相应的大小。这是为了确保随后的写入操作不会发生错误。
+
+在流被关闭前，更改不会写入到磁盘上实际的文件。更改首先会被写入到替代的临时文件。
+
+## 语法
+
+```js-nolint
+truncate(size)
+```
+
+### 参数
+
+- size
```

Comment:
It would be better to wrap this with `` ` `` (inline code), could you update the en-US content?

Reply:
https://github.com/mdn/content/pull/30109#issue-1983543300

Reply:
```suggestion
- `size`
```